zoofilter/01.basics/03.integration/docs.md

```diff
@@ -3,3 +3,156 @@ title: Integration
 taxonomy:
     category: docs
 ---
+
+## Search form
+
+ZOOfilter Search form is displayed by the *ZOOfilter Module* and controlled by the *ZOOfilter Plugin*. Currently is limited to search through one App and Type at the time.
+
+The Form inputs are composed by the Elements in the ZOOfilter layouts.
+
+To configure the search form please follow the next instructions:
+
+1. Go to the ZOO/gearwheel tab/<your app>
+2. Open the *ZOOfilter Search* Default layout for the Type you would like to create the Form for. 
```

Comment:
the doc theme doesn't support italics, instead we use bold. Eg. **ZOOfilter Search**. Change all of them and use wisely as too much bold text can be unproductive.